motorsd/test/test.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+
+#define SOCKET_PATH "/tmp/motorsd_socket"
+
+int main(int argc, char *argv[]) {
```

Comment:
Can u highlight somehow that this test represents a client connection to the server?